config/mail.php

```diff
@@ -0,0 +1,137 @@
+<?php
```

Comment:
I think we don't need to have this configuration file because Lumen still has access to all Laravel's default configuration files via the Illuminate namespace. 

If we have any modification to do on any of these configurations we can use the .env file.

Reply:
removed